scripts/update_schema_2.9.sql

```diff
@@ -0,0 +1,5 @@
+-- The remaining index on (id_run, position, tag_index)
+-- will provideeffective indexing for the columns
+-- (id_run, position).
+drop index iseq_pm_run_pos_index on iseq_product_metrics;
```

Comment:
I think we will get better performance with a single `ALTER TABLE` with multiple `DROP INDEX`.

```sql
ALTER TABLE iseq_product_metrics DROP INDEX iseq_pm_run_pos_index, DROP INDEX iseq_pr_metrics_lm_fk;
```

In devel they are both fast but there are less rows around.